### PR TITLE
SEO-174657-NET-MAUI-Badge-View-Eternallinking

### PR DESCRIPTION
**Pull request by Omanyadickson**
|Title|Description|
|--|--|
|Task Category| .NET MAUI Badge View
|Content Task Link/Mail Screenshot | https://dev.azure.com/Syncfusion-General/Content/_workitems/edit/764224/ |
|UX task|NA|
|External linking | PR - https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-badge-view/pull/4|
|Ticket/Task link | https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/175691/|
|Excel/SharePoint link | https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI---Money-Keywords.aspx|
 
| Keyword | Volume | Position | Reason for change | Changes Made |
| ---      | ---       | ---     | ---     | ---     |
|.net maui badge view|0-10 |5FT|To move FT to the first position |Changed the content alignment, improved some content|
|.net maui badge view example|0-10 |2UG |To improve the FT position|added in the md file |
|.net maui badge view control|0-10 |4FT |To move FT to the first position |Demo link added |
|.net maui badge view animation |0-10 |1UG |To move FT to the first position | need to add the keyword in the FAQ section|
|.net maui badge view customization|0-10 |1FT| To move FT to the first position | added new section|
 
 
Worked on 16 keywords in the .NET MAUI Badge View and am keeping track of all keyword details in the link below:-
https://syncfusion.sharepoint.com/:u:/r/sites/GH/SEO/SitePages/Keyword-Plan---.NET-MAUI-Badge-View.aspx

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # Getting Started with the .NET MAUI BadgeView Control
-A quick-start project that helps you integrate the Syncfusion Badge View control with a .NET MAUI application by adding the Syncfusion MAUI Core package from NuGet. This project contains simple code to apply predefined styles, icons, and other customizations.
-
-Documentation: https://help.syncfusion.com/maui/badge-view/getting-started
+The [.NET MAUI Badge View](https://www.syncfusion.com/maui-controls/maui-badge-view?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples) is a notification control that is used to notify users of new or unread messages, notifications, or status information. This article explains the steps required to configure the .NET MAUI Badge View control for .NET MAUI Badge Notifications and customize its elements.
```
Comment (Sharly-Atieno) [inline README.md]: Use as below:

The [.NET MAUI Badge View](https://www.syncfusion.com/maui-controls/maui-badge-view?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples) is a notification control that is used to notify users of new or unread messages, notifications, or status information. A quick-start project that helps you integrate the Syncfusion Badge View control with a .NET MAUI application by adding the Syncfusion MAUI Core package from NuGet. This project contains simple code to apply predefined styles, icons, and other customizations.

File: README.md [hunk 2]
```diff
@@ -52,4 +50,53 @@ If you are facing a path too long exception when building this example project,
...
+Eight predefined contextual color variants are available for badges, including primary, secondary, warning, error, and more.
+
+### Alignment 
+Align badges to their content in start, end, and center modes.
+
+### Icon
+The .NET MAUI Badge View control provides predefined notification symbols such as available, busy, away, delete, and more.
+
+### Customization
+Customize the MAUI Badge View control’s background color, text color, and font styles easily.
```
Comment (Sharly-Atieno) [inline README.md]: Need to interlink UG where possible.

File: README.md [hunk 3]
```diff
@@ -52,4 +50,53 @@ If you are facing a path too long exception when building this example project,
...
\ No newline at end of file
+Syncfusion has no liability for any damage or consequence that may arise by using or viewing the samples. The samples are for demonstrative purposes, and if you choose to use or access the samples, you agree to not hold Syncfusion liable, in any form, for any damage that is related to use, for accessing, or viewing the samples. By accessing, viewing, or seeing the samples, you acknowledge and agree Syncfusion’s samples will not allow you seek injunctive relief in any form for any claim related to the sample. If you do not agree to this, do not view, access, utilize, or otherwise do anything with Syncfusion’s samples.
+
+## Features and Benefits
+
+### Position
```
Comment (Sharly-Atieno) [inline README.md]: Update the topics the way they appear in live
image

File: README.md [hunk 4]
```diff
@@ -52,4 +50,53 @@ If you are facing a path too long exception when building this example project,
...
+Syncfusion's [.NET MAUI UI Controls](https://www.syncfusion.com/maui-controls?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples) library is the only suite that you will ever need to build an application since it contains over 40 high-performance, lightweight, modular, and responsive UI Controls in a single package. In addition to Radial Gauge, we provide popular .NET MAUI Controls such as [DataGrid](https://www.syncfusion.com/maui-controls/maui-datagrid?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples)[Charts](https://www.syncfusion.com/maui-controls/maui-cartesian-charts?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples), [Scheduler](https://www.syncfusion.com/maui-controls/maui-scheduler?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples), [ListView](https://www.syncfusion.com/maui-controls/maui-listview?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples), and [Excel Library](https://www.syncfusion.com/document-processing/excel-framework/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples).
+
```
Comment (Sharly-Atieno) [inline README.md:92]: Update the correct control here:
image

File: README.md (hunk 1 above)
Comment (Omanyadickson) [reply README.md]: Correction made

File: README.md (hunk 3 above)
Comment (Omanyadickson) [reply README.md]: Updated

File: README.md (hunk 4 above)
Comment (Omanyadickson) [reply README.md:92]: Updated the correct control.

File: README.md (hunk 2 above)
Comment (Omanyadickson) [reply README.md]: UG Link interlinked.

File: README.md [hunk 5]
```diff
@@ -52,4 +50,53 @@ If you are facing a path too long exception when building this example project,
...
+Eight predefined contextual color variants are available for badges, including primary, secondary, warning, error, and more.
+
+### Badge Alignment 
+Align badges to their content in start, end, and center modes.
+
+### Badge Icon
+The .NET MAUI Badge View control provides predefined notification symbols such as available, busy, away, delete, and more.
+
+### Badge Customization
+Customize the [.NET MAUI Badge View Customization](https://help.syncfusion.com/maui/badge-view/badge-customization) control’s background color, text color, and font styles easily.
```
Comment (Sharly-Atieno) [inline README.md]: Add UTM parameter after the links

Comment (Omanyadickson) [reply README.md]: UTM parameter added

File: README.md [hunk 6]
```diff
@@ -40,7 +38,7 @@ An Image, button, or label or any view can be added to the Badge View using the
 
 ## Project pre-requisites
 
-Make sure that you have the compatible versions of Visual Studio with .NET MAUI workloads and .NET SDK version in your machine before starting to work on this project. Refer to [System Requirements for .NET MAUI](https://help.syncfusion.com/maui/system-requirements).
+Make sure that you have the compatible versions of Visual Studio with .NET MAUI workloads and .NET SDK version in your machine before starting to work on this project. Refer to [System Requirements for .NET MAUI](https://help.syncfusion.com/maui/system-requirements?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples).
```
Comment (Sharly-Atieno) [inline README.md]: Why UTM parameter added here?

File: README.md [hunk 7]
```diff
@@ -52,4 +50,53 @@ If you are facing a path too long exception when building this example project,
...
+Eight predefined contextual color variants are available for badges, including primary, secondary, warning, error, and more.
+
+### Badge Alignment 
+Align badges to their content in start, end, and center modes.
+
+### Badge Icon
+The .NET MAUI Badge View control provides predefined notification symbols such as available, busy, away, delete, and more.
+
+### Badge Customization
+Customize the [.NET MAUI Badge View Customization](https://help.syncfusion.com/maui/badge-view/badge-customization?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples) control’s background color, text color, and font styles easily.
```
Comment (Sharly-Atieno) [inline README.md]: Need to interlink Customization only..

File: README.md (hunk 6 above)
Comment (Omanyadickson) [reply README.md]: UTM parameter removed

File: README.md (hunk 7 above)
Comment (Omanyadickson) [reply README.md]: Only Customization linked.

**Review by Sharly-Atieno — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**